beets/library.py

```diff
@@ -513,8 +519,10 @@ def write(self, path=None):
         else:
             path = normpath(path)
 
-        tags = dict(self)
-        plugins.send('write', item=self, path=path, tags=tags)
+        item_tags = dict(self)
+        if tags is not None:
```

Comment:
This could just be `if tags:` because an empty dict (which is also falsy) would have no effect.